niceday-api/index.js

```diff
@@ -45,10 +47,31 @@ function createNicedayApiServer() {
   return server;
 }
 
+function setupTokenRegeneration() {
+  const rule = new schedule.RecurrenceRule();
+  rule.hour = new schedule.Range(0, 23, 9);
+
+  const job = schedule.scheduleJob(rule, () => {
+    authSdk.login(THERAPIST_EMAIL_ADDRESS, THERAPIST_PASSWORD)
+      .then((response) => {
+        app.set('therapistId', response.user.id);
+        app.set('token', response.token);
+      })
+      .catch((error) => {
+        throw Error(`Error during authentication: ${error}`);
```

Comment:
Will it automatically retry if authentication fails?

Reply:
(Is that a feature of schedule/job?)

Reply:
No, this should not retry, but if the token expired, and sending a message fails as a consequence, then this will be regenerated. I hope that this will be enough